src/messages.ts

```diff
@@ -47,7 +47,7 @@ export type MismatchInfo<Actual, Expected> =
             K extends keyof Expected ? Expected[K] : never
           >
         }
-    : StrictEqualUsingBranding<Actual, Expected> extends true
+    : StrictEqualUsingBranding<Actual, Expected, DeepBrandOptionsDefaults> extends true
```

Comment:
this is temp - we should make sure we respect whatever options the `.branded` that's calling this has mebe??